fabric/0001-Integrate-tlcc-syscc.patch

```diff
@@ -0,0 +1,47 @@
+From 9d0105b879dd3cc048944d15aeadd84a973da5d0 Mon Sep 17 00:00:00 2001
+From: Marcus Brandenburger <bur@zurich.ibm.com>
+Date: Tue, 7 Apr 2020 15:07:09 +0200
+Subject: [PATCH] Integrate tlcc syscc
+
+Loads tlcc system chaincode on peer startup.
+
+Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>
+---
+ internal/peer/node/start.go | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/internal/peer/node/start.go b/internal/peer/node/start.go
+index 340d6da26..5fd9ebba3 100644
+--- a/internal/peer/node/start.go
++++ b/internal/peer/node/start.go
+@@ -74,6 +74,7 @@ import (
+ 	"github.com/hyperledger/fabric/core/scc/cscc"
+ 	"github.com/hyperledger/fabric/core/scc/lscc"
+ 	"github.com/hyperledger/fabric/core/scc/qscc"
++	tlcc "github.com/hyperledger/fabric/core/scc/tlcc"
```

Comment:
kind of weird it gets imported like this.

Reply:
shouldn't it also work without the `tlcc` prefix?

Reply:
right! will change